src/ethereum.ts

```diff
@@ -7,8 +7,7 @@ import cbor from 'borc';
 //@ts-expect-error - This third-party package is not typed properly
 import { TypedDataUtils } from 'eth-eip712-util-browser';
 import { keccak256 } from 'js-sha3';
-import { encode as rlpEncode } from 'rlp';
-import rlp from 'rlp-browser'; // [TODO: Deprecate]
+import rlp from 'rlp';
```

Comment:
Pls import just the encode function as before.

Reply:
7ce1ed8